x/stakeibc/keeper/unbonding_records.go

```diff
@@ -421,105 +415,98 @@ func (k Keeper) SweepAllUnbondedTokensForHostZone(ctx sdk.Context, hostZone type
 		//      2. the unbonding time is less than the current block time
 		//      3. the host zone is in the EXIT_TRANSFER_QUEUE state, meaning it's ready to be transferred
 		inTransferQueue := hostZoneUnbonding.Status == recordstypes.HostZoneUnbonding_EXIT_TRANSFER_QUEUE
-		validUnbondingTime := hostZoneUnbonding.UnbondingTime > 0 && hostZoneUnbonding.UnbondingTime < blockTime
+		validUnbondingTime := hostZoneUnbonding.UnbondingTime > 0 && hostZoneUnbonding.UnbondingTime < hostBlockTime
 		if inTransferQueue && validUnbondingTime {
-			k.Logger(ctx).Info(utils.LogWithHostZone(hostZone.ChainId, "  %v%s included in sweep", hostZoneUnbonding.NativeTokenAmount, hostZoneUnbonding.Denom))
+			k.Logger(ctx).Info(utils.LogWithHostZone(chainId, "  %v%s included in sweep",
+				hostZoneUnbonding.NativeTokenAmount, hostZoneUnbonding.Denom))
 
-			if err != nil {
-				errMsg := fmt.Sprintf("Could not convert native token amount to int64 | %s", err.Error())
-				k.Logger(ctx).Error(errMsg)
-				continue
-			}
-			totalAmtTransferToRedemptionAcct = totalAmtTransferToRedemptionAcct.Add(hostZoneUnbonding.NativeTokenAmount)
-			epochUnbondingRecordIds = append(epochUnbondingRecordIds, epochUnbondingRecord.EpochNumber)
+			totalSweepAmount = totalSweepAmount.Add(hostZoneUnbonding.NativeTokenAmount)
+			unbondingRecordIds = append(unbondingRecordIds, epochUnbondingRecord.EpochNumber)
 		}
 	}
 
-	// If we have any amount to sweep, then we can send the ICA call to sweep them
-	if totalAmtTransferToRedemptionAcct.LTE(sdkmath.ZeroInt()) {
-		k.Logger(ctx).Info(utils.LogWithHostZone(hostZone.ChainId, "No tokens ready for sweep"))
-		return true, totalAmtTransferToRedemptionAcct
-	}
-	k.Logger(ctx).Info(utils.LogWithHostZone(hostZone.ChainId, "Batch transferring %v to host zone", totalAmtTransferToRedemptionAcct))
```

Comment:
We don't need these messages in the logs any longer?  Seems useful but not critical

Reply:
Ah I see the same messages do still exist but in a different calling function

Reply:
yup exactly